document/src/main/java/com/ritense/document/service/DocumentDefinitionService.java

```diff
@@ -67,4 +67,5 @@ public interface DocumentDefinitionService {
 
     boolean isValidJsonPath(JsonSchemaDocumentDefinition definition, String jsonPathExpression);
 
+    void validateJsonPointer(String documentDefinitionName, String jsonPointer);
```

Comment:
Just a throught, not something to act on per say:

If we know an exception can occur (by design, in this case because the JsonPointer does not point at an existing property in the document definition) shouldn't we also make throwing the exception a part of the interface? So then it would become:

`void validateJsonPointer(String documentDefinitionName, String jsonPointer) throws ValidationException;`

Reply:
I've added it